### ISSUE TITLE
(Pre-)Deploy those Shiny apps from guest lecturers with a professional plan on ShinyApps.io

### ISSUE THREAD

**Issue by yihui** [rstudio-education/arm-workshop-rsc2019#32]
Jiena's app was on a free plan, which couldn't be viewed by 60+ students simultaneously. We should deploy such apps with a professional plan by ourselves in the future.

**Comment by apreshill** [comment]
Yes I spoke with Carl about this yesterday- if I can get the code I will put it up today!

**Comment by apreshill** [comment]
@jienagu if you want to link me to source code for the app, I can deploy for you. I don't see it on GitHub though.
